specifications/api/swagger.yaml

```diff
@@ -4437,6 +4479,19 @@ paths:
         - *parametersWalletId
       responses: *responsesGetUTxOsStatistics
 
+  /wallets/{walletId}/utxo:
+    get:
+      operationId: getWalletUtxoSnapshot
+      tags: ["Wallets"]
+      summary: A snapshot of the wallet's UTxO set
+      description: |
+        Generate a snapshot of the wallet's UTxO set.
```

Comment:
Most endpoints have a `status: stable` note.
Since this is intended to be used for debugging, better to not imply stability - perhaps add something like `status: unstable`.

Reply:
> Most endpoints have a status: stable note.
> Since this is intended to be used for debugging, better to not imply stability - perhaps add
> something like status: unstable.

Good point!